src/main/connector/semconv/Database.yaml

```diff
@@ -87,3 +87,11 @@ metrics:
     description: "The number of database threads categorized by their state (cached, connected, created, or running)."
     type: Gauge
     unit: "{thread}"
+  db.server.operations.wait_time:
```

Comment:
`wait_time` is weird and sounds specific. To be aligned with system semconv, we should use `db.server.operation_time` (or `db.server.disk.operation_time` if the metric is specific to disk IO activities).